packages/office-ui-fabric-react/src/components/Facepile/Facepile.tsx

```diff
@@ -79,16 +88,15 @@ export class Facepile extends React.Component<IFacepileProps, {}> {
   }
 
   private _getElementWithoutOnClickEvent(personaControl: JSX.Element, persona: IFacepilePersona, index: number): JSX.Element {
-    return <div
+    return <button
       { ...getNativeProps(persona, divProperties) }
```

Comment:
since you changed this to button, may want to get native `buttonProperties`. (You can change the import.)